src/um/wtsapi32.rs

```diff
@@ -0,0 +1,959 @@
+ // Copyright © 2015-2018 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::minwindef::{BOOL, BYTE, DWORD, LPDWORD, MAX_PATH, PBOOL, PULONG, ULONG, USHORT};
+use shared::windef::HWND;
+use um::winnt::{
+    CHAR, HANDLE, LARGE_INTEGER, LONG, LPSTR, LPWSTR, PCHAR, PSECURITY_DESCRIPTOR, PSID, PSTR,
+    PVOID, PWSTR, SECURITY_INFORMATION, STANDARD_RIGHTS_REQUIRED, WCHAR
+};
+pub const WTS_CURRENT_SERVER: HANDLE = 0isize as HANDLE;
+pub const WTS_CURRENT_SERVER_HANDLE: HANDLE = 0isize as HANDLE;
```

Comment:
The `isize` literal suffix shouldn't be needed here since it isn't negative and it is being cast to a pointer.

Reply:
removed